package-scripts/datadog-agent/preinst

```diff
@@ -46,10 +46,27 @@ if [ "$DISTRIBUTION" != "Darwin" ]; then
       useradd -r -M -g dd-agent -d $INSTALL_DIR -s /bin/sh \
         -c "Datadog Agent" dd-agent
     # Stop the old agent before installing
-    if [ -f "/etc/init.d/datadog-agent" ]; then
```

Comment:
I think I meant to copy something and cut instead, but not really sure how I did it, either way it's back.